### ISSUE TITLE
2.1.1 Captains and 4.3 Challenges - who can challenge?

### ISSUE THREAD

**Issue by jswingle** [gryphonshafer/Quizzing-Rule-Book#123]
Sections 2.1.1 and 4.3 specify that the captain and co-captain may both challenge, which is a recent change. Originally the rulebook allowed for only the captain to challenge (with the co-captain becoming acting captain, of course, in the event the captain left the platform.)

I don't know why this change was made, and I've heard a lot about rolling the rule back to only allowing the captain to challenge. But perhaps we need to go the other direction: what if the _entire team_ was allowed to consult together before one quizzer presents the challenge to the officials?

The major con I see to this is that it would take a bit of additional time, but we could keep this reasonable by putting a time limit on the consultation period (say, 60 seconds).

As for the pros, I see several: involving more quizzers in the process of understanding the rules, building up teamwork and team communication, and challenges will tend towards being more informed and stronger as the other quizzers can add weight to the argument.

There are a lot of variations we could add to this rule. For example, maybe the team is allowed to consult, but the captain (or co-captain?) still has to be the one to initiate the challenge. I would imagine the other teams should not be allowed to consult, but maybe they would be allowed to before presenting a rebuttal.

**Comment by josiah-leinbach** [comment]
If I recall correctly, the logic given for letting co-captains challenge had to do with specialization, namely that if the team captain studied REFs but not FTVs (which the co-captain did), then both should be allowed to challenge since one may know something the other does not.

Personally, I am not a fan of co-captains challenging for several reasons. First, it makes the role of "captain" irrelevant since both captain and co-captain have the same privileges (calling timeouts and challenging). So functionally, there are two co-captains, and they comprise half of the quizzers on the bench, thereby slightly cheapening their authority. Second, having two leaders with _de facto_ equal authority is a recipe for confusion on several levels. Given the rule about no communicating prior to or during a challenge, I have seen many issues in real quizzing where the functional co-captains are trying to figure out who should challenge by exchanging looks, etc. Then either (1) one of them does and no foul is called, thus violating the rulebook on that point, or else (2) a foul is called and they lose the ability to challenge because they had to communicate about who was challenging and were prohibited from doing so. When everyone knows that one person and only one person has this authority, things flow more smoothly. 

Plus, we run into issues with overruled challenges. Let's say the captain (a very strong quizzer) makes a very good challenge but nevertheless and  is overruled, but later in the same quiz the co-captain (a less strong quizzer) makes a very weak challenge and is overruled. The team loses points there because the captain, who knew better, could not stop his co-captain from making a bad challenge. Now of course, the way to prevent this would be for a team to understand that the captain has final authority. But now we have reestablished the hierarchy in practice without supporting it in law. 

Thus, the rule is best reverted to the original that the captain and only the captain can challenge.

**Comment by scottpeterson** [comment]
I couldn't care less about preserving any "authority" for the captain. They're all quizzers.

**Comment by scottpeterson** [comment]
I do agree that allowing > 1 quizzer per team increases the amount of potential "consulting" that the QM will have to judge if it was to the level of a foul.  

That potential exists now of course. (you can't consult with teammates or your coach, even if none of them can challenge). But it's definitely increased when > 1 quizzer from the same team may be wanting to challenge. 

Personally I'm fine with that. A look or nod is totally fine for me. But that's not objectively defined.

**Comment by jswingle** [comment]
Yes, this would be one way of getting rid of that rulebook "bug" about communicating between team members but letting more than one quizzer per team challenge at a time. That should get resolved one way or another (I presume there's already an open issue about it) but this is just one idea that could resolve it.

**Comment by jswingle** [comment]
From a quizzing culture standpoint, there is some advantage to providing a natural "leadership role" baked-in to the competition. It helps develop leadership skills, from what I've observed and through personal experience. It's not an indispensable aspect of quizzing, but I'm not indifferent to it either. 

That said, the role of the captain could change dramatically and yet this bonus to quizzing culture would still be there.

**Comment by ARMediting** [comment]
I am concerned that giving teams time to consult on a challenge would add a lot more dead time in a quiz. One type of quizzer you sometimes have on a team is a passionate, somewhat knowledgeable kid who's always up for a challenge. A coach might not make this quizzer a captain because they'll attempt too many poor challenges. I feel like if you let any quizzer challenge, you'll still have coaches telling certain quizzers not to challenge, and if you allow consultation, you'll see some quizzers putting a lot of pressure on a captain in a potentially emotionally fraught scenario (e.g., a quizzer got an error and thinks the captain should challenge for him when the captain knows it wouldn't help).

**Comment by jttower** [comment]
I agree with @ARMediting that sometimes you get a quizzer on a team who is "challenge-happy".  I think we need to either stick with allowing the co-captains to challenge, or move back to just allowing the captain to challenge.  I don't think we should allow discussion or material consultation before or during a challenge.

**Comment by scottpeterson** [comment]
well stated Alex.

**Comment by jswingle** [comment]
It seems like my idea for allowing group consultation hasn't been popular, so let's focus on getting rid of the co-captain's ability to challenge and rebut. This rule was changed a few years back, to the surprise and confusion of everyone I personally know in quizzing. Does anyone know why this rule was changed at all? What was the motivation?

**Comment by scottpeterson** [comment]
While I agree with the bulk of Alex's latest post, I'll pick a nit here: " coach might not make this quizzer a captain because they'll attempt too many poor challenges."   IMO, that should be handled by the coach, not by the rulebook.

**Comment by scottpeterson** [comment]
I don't recall why the rule was changed. Perhaps @ZacharyTinker knows?

Really very little changed for me as a QM when this rule changed. I think I started directing "rebuttal?" to a team rather than to the captain. I think that's all that changed for me?   I don't recall encountering increased issues regarding consulting/fouls because of the co-captain addition.

**Comment by scottpeterson** [comment]
Agree with all of that. 

But we'd then need to define what is required of a QM, information wise. Otherwise less confident QMs will intentionally withhold information to reduce the "surface area" of potential challenges. (this is already a problem).

**Comment by scottpeterson** [comment]
I'm only against quizzer consultation because I think it would be a massive time sink.

**Comment by gryphonshafer** [comment]
I agree with the consultation time concern. What if each team has a maximum of 30 seconds? I'm not saying this is a good idea, but it's a simple idea that might address the concern.

I also agree with the concern about QMs withholding information. This would be more difficult to address. Maybe something like: "Prior to issuing a challenge, quizzers may ask the QM for material and/or ruling clarification. QMs will not withhold information."

**Comment by jswingle** [comment]
It's worth pointing out that team consultation is a feature of quite a few other quizzing programs. That is by no means a reason why we should do it ("but all the other nations have a king!") but my point being that it's not a crazy novel idea. Other programs will literally get a time period to bust out the quiz material and rulebook so they can reference specific things in the text and rules.

**Comment by scottpeterson** [comment]
I think Gryphon's sentiment is key.  While knowledge of the rules (and material knowledge for use during challenges) can be a competitive advantage, the competition isn't seeking to test quizzers on their rule book knowledge. Challenges and protests exist to give every possible chance to get the right end ruling. Consultation, multiple challengers, rule book consultation by quizzers/coaches---all of that would help meet the goal of getting to the right end ruling.

**Comment by jswingle** [comment]
> > I'm fine with the captain and co-captain exchanging glances, if we can word it in a way that isn't too cumbersome, but I don't want there to be a lengthy discussion prior to challenging.

This is really the heart of my concern with the change to allowing co-captains to challenge at the same time as captains. It doesn't make any sense that we allow two different quizzers to challenge/rebut but don't allow any sort of nonverbal communication during the challenge event. This is a straight-up bug in my opinion. Simplest fix is to just revert it so that only captains can challenge, and upon them leaving the stage the baton gets passed to someone else, either the co-captain or a designated third in the rare instance of two err outs.

**Comment by ARMediting** [comment]
Just curious, would all of the sentiments presented above also apply to rebuttals (minus someone initiating the challenge)? Because you pretty much have to have some communication about who handles that. I guess I'm thinking along the same lines as Jeremy, that our biggest danger is the potential time sink. If the original goal of the co-captain is to ensure someone's always in who can handle challenges, then add a rule to let coaches appoint a co-co-captain during a quiz if needed. Or some sort of armband system, where it can be handed off.

**Comment by jswingle** [comment]
A lack of ability to challenge any further could also be seen as simply "that's the penalty your team gets for having your captain and co-captain err out." A coach can still protest in such circumstances, so it's not like the team loses all representation.

That said, I don't think there's much an issue with just having the coach be asked to appoint a 3rd after the co-captain errs out.

If no one is in favor of my wacky ideas to have everyone be able to challenge and consult with each other, then I think the next best option is to revert it to the way it used to be and have only one quizzer able to challenge and rebut at any given time.

**Comment by levikoral** [comment]
I think that the captain and the so called "co-captain" should be allowed to challenge but if both of them are out of the quiz, then there should be another captain appointed.

**Comment by jswingle** [comment]
Many ideas have been thrown around about challenge procedure -- seems there's not yet much consensus.

I would like to move this issue in one of two directions, if others are willing: (1) remove the ability of the co-captain to challenge until the captain has left the stage, (2) allow the captain and co-captain to consult in some manner briefly before presenting the challenge.

I'm happy with both options. I don't think the exact procedure selected will make too much of a difference -- I'm more interested in fixing the current rulebook bug concerning communication between quizzers.

**Comment by josiah-leinbach** [comment]
I would argue for the former option for a few reasons, some of which I have articulated above:

1. If the captain and co-captain can both challenge, then either the term "captain" or "co-captain" is meaningless. They are either both captains or both co-captains with equal authority at that point. That authority is watered down when half of the sitting quizzers have the right to challenge and thus hurts team leadership. 

2. Allowing the captain and co-captain to consult actually compounds the problem I noted earlier, namely when the they disagree about whether to challenge. For example, if one wants to challenge and the other does not, you have an impasse. Should they be allowed to debate the matter between themselves? That just adds time, tension, and confusion to the quiz. Do you defer to the quizzer who initiated the challenge? Unless they were persuaded in that (presumably brief) conversation with their captain/co-caption, they will challenge anyway and the whole point of consulting is irrelevant. Should consulting be _required_? It would have to be, or else the whole point of having two people with this authority to challenge is pointless and we would be back to the problem of communication between quizzers. But if you require it, that just produces the very circumstances for the impasse to begin with. What if the co-captain has a very weak challenge and challenges against the captain's wishes and the challenge is overruled? Now the captain's hands are tied for the rest of the quiz, because if they challenge and are overruled, the team loses points, even though they did not consent to the earlier decision to challenge.

3. Section 3.1.1 requires teams to name a captain _and_ co-captain, which leaves open the possibility that a team is forced to name a quizzer who is not prepared for the responsibilities of sharing that _de jure_ equal authority. This has two possible results, both of which are bad. On the one hand, if they recognize that they are not ready for that role, they will likely just defer to the captain, so consulting becomes at best redundant (because, as I explained above, it would need to be required) and at less-than-best something which in practice will be routinely disregarded (i.e. the captain will just make the challenge anyway without consulting).

Of course, someone could argue that we can just make additional rules to solve these problems. We can rename both persons as "captains". We can set time limits about how long the captains can consult. We can create a procedure to arbitrate a disagreement between them. Yet these are all ad hoc solutions to a problem that is in fact _created_ by allowing co-captains to challenge in the first place. Co-captains challenging came into the rulebook for inauspicious reasons (i.e. to make narrow, list-based specialization less of a liability for the team captain), and it not only serves no positive good for quizzing but it creates active problems. 

Therefore, I propose we strike "co-captain" and "or co-captain" from the relevant passages for this issue in 3.1.1, 5.3, 5.5, and 5.6.

**Comment by gryphonshafer** [comment]
> Co-captains challenging came into the rulebook for inauspicious reasons (i.e. to make narrow, list-based specialization less of a liability for the team captain), and it not only serves no positive good for quizzing but it creates active problems.

Co-captains were created so teams could still challenge when the captain has erred out of the quiz.

**Comment by josiah-leinbach** [comment]
@gryphonshafer Yes. I should have clarified to say that I meant co-captains being allowed to challenge _while captains are still in the quiz_. I certainly believe co-captains should be retained as an entity, but that they may not exercise the same privileges as a captain unless that captain has erred out. In other words, I propose a return to the prior version of this current rule.

**Comment by gryphonshafer** [comment]
The problem with "co-captain" is that it doesn't actually solve the problem for which it was designed. If the captain and co-captain are erred out, there needs to be a co-co-captain. It's turtles all the way down.

Crazy idea: What would be the harm in eliminating the captain and co-captain language entirely from the rule book and allowing any quizzer on a team to challenge (sans consulting if we want to keep it that way)? Coaches would then be free to organize their teams' platform leadership strategy however they felt was best.

**Comment by jttower** [comment]
I think allowing the captain and co-captain to challenge does allow the opportunity for a more accurate challenge, since one may feel more confident than the other on a question type. I am in favor of calling both of them “co-captains”, since that is what they are. As far as solving the “communication” bug, I think defining what constitutes communication between quizzers/coaches, etc, would solve this issue. Allowing non-verbal communication like a nod or eye contact, etc.

**Comment by jswingle** [comment]
I think the categories of captain and co-captain are totally fine as is, though I wouldn't be opposed to changing the language of "co-captain" to something that makes more sense, like "alternate captain" or "secondary captain"

@gryphonshafer The rulebook already specifies that a coach can designate a 3rd quizzer to initiate challenges and respond to them if both the captain and co-captain err out, so I don't see an issue here. We ask coaches to designate a captain and co-captain before the quiz because the occurrence of a captain erring out is _super common_, but to require the coaches to designate a 3rd doesn't become a necessity until the _super rare_ occurrence of both the captain and co-captain erring out (or the captain quizzing out, getting replaced by the sub, and then the co-captain erring out). For practical reasons, it makes sense to have a category for the co-captain, and then to have a rule that covers the super rare circumstances.

I don't think "any quizzer can challenge" works unless we implement the idea of quizzers consulting with each other like I've enumerated above. It either needs to be only one quizzer allowed per team at a time, or allow the quizzers to communicate with each other before lodging a challenge. Otherwise we end up with awkwardness deciding who actually gets to challenge or not.

**Comment by ARMediting** [comment]
[Note: I updated this issue title to have the correct current location of the involved rules.]

Jeremy's comment seems to have captured the heart of this issue:

> Many ideas have been thrown around about challenge procedure -- seems there's not yet much consensus.
> 
> I would like to move this issue in one of two directions, if others are willing: (1) remove the ability of the co-captain to challenge until the captain has left the stage, (2) allow the captain and co-captain to consult in some manner briefly before presenting the challenge.
> 
> I'm happy with both options. I don't think the exact procedure selected will make too much of a difference -- I'm more interested in fixing the current rulebook bug concerning communication between quizzers.

The third option would be to change nothing, but this leaves the "bug" mentioned. My thought is it's only a concern if the two captains on a team _are_ talking before an appeal. They don't have to. If they both stand up to appeal, the QM can decide how to handle it, either by letting them confer or choosing one.

I'm also fine with removing the ability of a co-captain to appeal. Being a captain should come with more demands of material knowledge and rule knowledge (to address the comment of "But what if it's a specialty that's not their type?"). And yes, the appeal process is more about following the rules than anything, but if we want to balance that with keeping the quiz moving, more of that responsibility should fall to the officials than anyone.
